libs/components/lookup/src/lib/modules/selection-modal/selection-modal.component.ts

```diff
@@ -288,4 +304,20 @@ export class SkySelectionModalComponent implements OnInit, OnDestroy {
       this.isSearching = false;
     }
   }
+
+  #announceSelectionState(
+    selectedItemCount: number,
+    displayedItemCount: number
+  ): void {
+    this.#resourcesSvc
+      .getString(
+        'skyux_lookup_show_more_displayed_items_updated',
+        selectedItemCount.toString(),
+        displayedItemCount.toString()
+      )
+      .pipe(take(1))
+      .subscribe((resourceString) => {
+        this.#liveAnnouncerSvc.announce(resourceString);
+      });
+  }
```

Comment:
the code looks good but the unit test file does not seem to test this block of code from what i can tell? There is no state change unit test that checks what live announcer does

Reply:
This always gets called on initialization through the search function. Added a spy to the announcer service to validate it